### PR TITLE
Updated the changelog with upcoming 0.9.0 changes

### PR DESCRIPTION
**Pull request by kolloch**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -5,6 +5,56 @@
...
+
+#### Restrict applications to certain Mesos roles
+
+Prior Marathon versions already support registering with a `--mesos_role`. This causes Mesos to offer resources
+of the specified role to Marathon in addition to resources without any role designation ("*").
+Marathon would use resources of any role for tasks of any app.
+
+Now you can specify which roles Marathon should consider for launching apps per default via the
+`--default_accepted_resource_roles` configuration argument. You can override the default by specifying
+a list of accepted roles for you app via the `"acceptedResourceRoles"` attribute of the app definition.
```
Comment (philipnrmn) [inline changelog.md]: s/you app/your app/

File: changelog.md [hunk 2]
```diff
@@ -13,9 +63,22 @@ on task launch.
...
+
+The UI now uses relative URL paths making it easier to run Marathon behind a reverse proxy.
+
+#### Restrict the number of versions by default
+
+In Marathon, most state is versioned. This includes app definitions and group definitions. Marathon already allowed
+restricting the number of versions that are kept by `--zk_max_versions` but you had to specify that explicitly.
 
-We changed the format of the `http_endpoints` parameters from a
+Since some of our users where running into problems with too many versions, we decided to restrict to
```
Comment (philipnrmn) [inline changelog.md]: s/where/were

**Comment by philipnrmn** [comment]
Not part of your changes, but s/follwing/following

**Comment by philipnrmn** [comment]
OK, otherwise all good :)

**Comment by kolloch** [comment]
Thanks for the review. I updated it accordingly.

**Pull request merged**